src/redux/actions/cellSets/reorderCellSet.js

```diff
@@ -0,0 +1,70 @@
+import { CELL_SETS_REORDER } from 'redux/actionTypes/cellSets';
+
+import fetchAPI from 'utils/fetchAPI';
+import endUserMessages from 'utils/endUserMessages';
+import pushNotificationMessage from 'utils/pushNotificationMessage';
+import { isServerError, throwIfRequestFailed } from 'utils/fetchErrors';
+
+const reorderCellSetJsonMerger = (cellSetKey, newPosition, cellClassKey) => (
+  [{
+    $match: {
+      query: `$[?(@.key == "${cellClassKey}")]`,
+      value: {
+        children: [
+          {
+            $match: {
+              query: `$[?(@.key == "${cellSetKey}")]`,
+              value: {
+                $move: newPosition,
+              },
+            },
+          },
+        ],
+      },
+    },
+  }]
+);
+
+const reorderCellSet = (
```

Comment:
This action creator/action combo is replacing the one of `updateCellSetHierachy` because reordering the cell sets was the only actual usage `updateCellSetHierachy` was receiving